Docs/CHANGELOG.md

```diff
@@ -121,6 +121,7 @@
     - Added support to use a non-CARLA OpenDRIVE map (instead of CARLA towns)
     - Added support for TimeOfDay tag
     - Added support for scenarios with no actors
+    - Added support for TimeToCollisionCondition with freespace.
```

Comment:
We are technically also supporting freespace for TimeHeadwayCondition now. Please update it. Also, the docs have to be updated. At *Docs/openscenario_support.md* search for these two conditions and remove the warning saying that freespace is not supported